cypher-shell/src/test/java/org/neo4j/shell/state/BoltStateHandlerTest.java

```diff
@@ -105,4 +106,49 @@ public void shouldExecuteInSessionByDefault() throws CommandException {
         assertEquals("Transaction should be the runner",
                 boltStateHandler.getCurrentSession(), boltStateHandler.getStatementRunner());
     }
+
```

Comment:
If these tests can be changed with Mockito then we can remove a bunch of stubbed classes like TestTransaction, TestDriver, TestSession etc..